wgpu-types/src/lib.rs

```diff
@@ -560,6 +560,13 @@ bitflags::bitflags! {
         ///
         /// This is a native-only feature.
         const TEXTURE_COMPRESSION_ASTC_HDR = 1 << 42;
+        /// Allows shaders to acquire the FP16 ability
```

Comment:
Could we also note that this isn't supported in naga, only through spir-v passthrough right now.